pkg/database/sql.go

```diff
@@ -92,7 +92,11 @@ func (d *db) VerifiableSQLGet(req *schema.VerifiableSQLGetRequest) (*schema.Veri
 		}
 	}
 
-	tx := d.st.NewTxHolder()
+	tx, err := d.allocTx()
```

Comment:
it might be moved some lines below, right before its usage

Reply:
Fixed in 92492aa9.